lib/vsc/administration/user.py

```diff
@@ -391,22 +417,32 @@ def populate_home_dir(self):
 
         Does not overwrite files that may contain user defined content.
         """
+        try:
+            storage = self.institute_storage[VSC_HOME]
+        except KeyError:
+            logging.exception("Trying to access non-existent institute storage: %s", VSC_HOME)
```

Comment:
Trying -> Failed

perhaps move this code block in some method instead of all the copy/paste?

Reply:
done in https://github.com/hpcugent/vsc-administration/pull/134/commits/d9d2f11e38cfec377807ac56339b0498d1b1400d